src/data/markdown/translated-guides/en/03 Results output/200 Real-time/00 Prometheus remote write.md

```diff
@@ -22,7 +22,7 @@ refer to [Cloud to Prometheus RW](/cloud/integrations/prometheus-remote-write/).
 
 </Blockquote>
 
-## About metrics mapping
+## Metrics mapping
```

Comment:
I think this title has been added by @MattDodsonEnglish so I would keep it.

Reply:
Previously, this section includes more content. Some previous sections are now separated on their own. 

With those changes, now `Metrics mapping` seems correct to me.